ci/release/update-version.sh

```diff
@@ -100,29 +124,20 @@ sed_runner "/^set(cuopt_version/ s/[0-9][0-9].[0-9][0-9].[0-9][0-9]/${NEXT_FULL_
 # Update nightly
 sed_runner 's/'"cuopt_version: \"[0-9][0-9].[0-9][0-9]\""'/'"cuopt_version: \"${NEXT_SHORT_TAG}\""'/g' .github/workflows/nightly.yaml
 
-# Update Service build
-sed_runner 's/'"nvcr.io\/j9mrpofbmtxd\/test\/cuopt:[0-9][0-9].[0-9][0-9]\(\.arm\)\?"'/'"nvcr.io\/j9mrpofbmtxd\/test\/cuopt:${NEXT_SHORT_TAG}\1"'/g' .github/workflows/service_nightly.yaml
-sed_runner 's/'"nvcr.io\/0616513341838337\/cuopt:[0-9][0-9].[0-9][0-9]\(\.arm\)\?"'/'"nvcr.io\/0616513341838337\/cuopt:${NEXT_SHORT_TAG}\1"'/g' .github/workflows/service_nightly.yaml
-
-# Update README.md
-sed_runner "s/\(cuopt-server-cu12==\)[0-9]\+\.[0-9]\+\.\\*/\1${PY_NEXT_SHORT_TAG}.\*/g" README.md
-sed_runner "s/\(cuopt-sh-client==\)[0-9]\+\.[0-9]\+\.\\*/\1${PY_NEXT_SHORT_TAG}.\*/g" README.md
-sed_runner 's/cuopt-server=[0-9][0-9].[0-9][0-9] cuopt-sh-client=[0-9][0-9].[0-9][0-9] python=[0-9].[0-9][0-9] cuda-version=[0-9][0-9].[0-9]/cuopt-server='${NEXT_SHORT_TAG}' cuopt-sh-client='${NEXT_SHORT_TAG}' python=3.12 cuda-version=12.8/g' README.md
-sed_runner 's|cuopt:[0-9]\{2\}\.[0-9]\{1,2\}\.[0-9]\+\(-cuda12\.8-\)\(py[0-9]\+\)|cuopt:'"${DOCKER_TAG}"'\1\2|g' README.md
-
 # Update Helm chart files
 sed_runner 's/\(tag: "\)[0-9][0-9]\.[0-9]\+\.[0-9]\+\(-cuda12\.8-py3\.12"\)/\1'${DOCKER_TAG}'\2/g' helmchart/cuopt-server/values.yaml
 sed_runner 's/\(appVersion: \)[0-9][0-9]\.[0-9]\+\.[0-9]\+/\1'${DOCKER_TAG}'/g' helmchart/cuopt-server/Chart.yaml
 sed_runner 's/\(version: \)[0-9][0-9]\.[0-9]\+\.[0-9]\+/\1'${DOCKER_TAG}'/g' helmchart/cuopt-server/Chart.yaml
 
-DEPENDENCIES=(
-  libcuopt
-  cuopt
-  cuopt-mps-parser
-)
-
-for DEP in "${DEPENDENCIES[@]}"; do
-  for FILE in python/*/pyproject.toml; do
-    sed_runner "s/\(${DEP}==\)[0-9]\+\.[0-9]\+/\1${PY_NEXT_SHORT_TAG}/" "${FILE}"
-  done
+for FILE in .github/workflows/*.yaml; do
+  sed_runner "/shared-workflows/ s/@.*/@branch-${NEXT_SHORT_TAG}/g" "${FILE}"
+  # CI image tags of the form {rapids_version}-{something}
+  sed_runner "s/:[0-9]*\\.[0-9]*-/:${NEXT_SHORT_TAG}-/g" "${FILE}"
 done
+
+# CI for RAPIDS
+sed_runner 's/'"DEPENDENT_PACKAGE_VERSION=\"[0-9][0-9].[0-9][0-9]\""'/'"DEPENDENT_PACKAGE_VERSION=\"${NEXT_SHORT_TAG}\""'/g' ci/build_cpp.sh
+sed_runner 's/'"DEPENDENT_PACKAGE_VERSION=\"[0-9][0-9].[0-9][0-9]\""'/'"DEPENDENT_PACKAGE_VERSION=\"${NEXT_SHORT_TAG}\""'/g' ci/build_python.sh
+
```

Comment:
```suggestion
```

You already have `RAPIDS_VERSION` in a file in the repo. Instead of doing this:

https://github.com/NVIDIA/cuopt/blob/0845a21e0a867babdce73325d1bcf748a32ca9d4/ci/build_cpp.sh#L35

How about doing this in those scripts?

```shell
DEPENDENT_PACKAGE_VERSION=$(cat ./RAPIDS_VERSION)
export DEPENDENT_PACKAGE_VERSION
```

That way, you don't need to update this in shell scripts... just drive everything through those `VERSION` and `RAPIDS_VERSION` files.

That'd mean fewer updates, less risk of inconsistencies, and an easier-to-understand configuration flow.